aptos-move/aptos-gas/src/aptos_framework.rs

```diff
@@ -40,6 +40,7 @@ crate::natives::define_gas_parameters_for_natives!(GasParameters, "aptos_framewo
     [.ristretto255.basepoint_double_mul, "ristretto255.basepoint_double_mul", 1],
 
     [.ristretto255.point_add, "ristretto255.point_add", 1],
+    [.ristretto255.point_clone, "ristretto255.point_clone", 1],
```

Comment:
And what about adding new gas parametes? @davidiw, @wrwg, @vgao1996?

Reply:
Adding a new native function will break everything sadly... See my earlier comment

Reply:
Uhm, you mean it's backwards compatible **but** needs careful deployment, or else it "breaks everything"?